src/views/Nft/market/Profile/components/ActivityHistory/ActivityPrice.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { Flex, BinanceIcon, Text } from '@pancakeswap/uikit'
+import { multiplyPriceByAmount } from 'utils/prices'
+import { useTranslation } from 'contexts/Localization'
+
+const ActivityPrice = ({ bnbBusdPrice, price }) => {
+  const { t } = useTranslation()
+  const priceInUsd = multiplyPriceByAmount(bnbBusdPrice, price)
+
+  return (
+    <Flex flexDirection="column" alignItems="flex-end">
+      {price ? (
+        <>
+          <Flex justifySelf="flex-start" alignItems="center">
+            <BinanceIcon width="12px" height="12px" mr="4px" />
+            <Text bold>{price}</Text>
+          </Flex>
+          <Text fontSize="12px" color="textSubtle">
+            {`(~${t('$%amount%', {
```

Comment:
Doesn't this result in two $$?

Reply:
I don't think so, first $ is the template string variable thingy `${x}`
But in bunch of other places we do have additional dollar sign here, e.g. CostLabel
```javascript
<Flex alignItems="center" {...props}>
      {priceInUsd > 0 && (
        <Text fontSize="12px" color="textSubtle">{`($${priceInUsd.toLocaleString(undefined, {
          minimumFractionDigits: 2,
          maximumFractionDigits: 2,
        })})`}</Text>
      )}
      <BNBAmountLabel amount={cost} />
    </Flex>
```

Reply:
Haha you're right.